### PR TITLE
Fredkiss3/eng2 31 homepage design improvements

### PR DESCRIPTION
**Pull request by fredk3**
This PR adds the new design improvements for the homepage.

### CODE REVIEW

File: apps/web/tailwind.config.cjs [hunk 1]
```diff
@@ -81,6 +81,14 @@ module.exports = {
         slideRightAndFade:
           "slideRightAndFade 400ms cubic-bezier(0.16, 1, 0.3, 1)",
       },
+      fontSize: {
```
Comment (lzrscg) [inline apps/web/tailwind.config.cjs:84]: isn't it better to rely on rem? 

https://austingil.com/px-or-rem-in-css/

File: apps/web/styles/globals.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 @layer base {
   :root {
-    font-size: 14px;
```
Comment (lzrscg) [inline apps/web/styles/globals.css:7]: oh i didn't realize this was 14 the whole time. i wonder how this change effects other parts of the app. i'll look into it

Comment (fredk3) [reply apps/web/styles/globals.css:7]: The default font size of the browser is 16px and usually rem sizes are determined in relation with that, it is what Figma also uses for determining rem sizes.

File: apps/web/tailwind.config.cjs (hunk 1 above)
Comment (fredk3) [reply apps/web/tailwind.config.cjs:84]: Yeah, I'll change that.

File: apps/web/styles/globals.css (hunk 2 above)
Comment (fredk3) [reply apps/web/styles/globals.css:7]: I'll just remove the default `font-size`.

**Comment by lzrscg** [comment]
Changing the base height expanded the spacing on the block page:
https://share.cleanshot.com/t7bCTnR0

Is there any way we can change the base height on the homepage now. Then we can change the base height on the block page later? We want the base height to be 16px everywhere. However, that will take a lot of work.

**Comment by fredk3** [comment]
> @lzrscg  Is there any way we can change the base height on the homepage now. Then we can change the base height on the block page later? We want the base height to be 16px everywhere. However, that will take a lot of work.

You mean the base font-size ? If so, it is already done, I changed the font size on the homepage and noticed that the text was way too big, so I adapted the text of the block page to better match what is in the design.

File: apps/web/styles/globals.css (hunk 2 above)
Comment (fredk3) [reply apps/web/styles/globals.css:7]: done.

**Comment by fredk3** [comment]
I think i need to also adapt the spacing & padding between items on the block page.

**Comment by lzrscg** [comment]
whenever we use `rem` for sizing, it is relative to the base size.

this is good because then its adjustable for people with different browser settings. however, since we changed it from 14 -> 16, then it enlarges everything

setting the font size to px is a partial solution, but makes the font not scale with browser settings. since spacing is mostly in rem and font is in px, the font is currently remaining the same size whereas the spacing is scaling. 

so we could fix the spacing, but that might be a bigger task. if we just leave the base size the same on the block page, then we don't need to do all of that. we can just adjust the base size of the homepage and then fix the block page later.

**Comment by fredk3** [comment]
I already started to modify the block page, I can finish with the spacing pretty quickly.

**Pull request merged**